doc/development/01docker+local.md

```diff
@@ -0,0 +1,25 @@
+# Docker + local
+
+Run the database in Docker and the frontend and backend locally.
+
+## Install Docker
+
+[Install Docker](https://docs.docker.com/engine/install/) for your host OS.
+
+## Installing Node 20 + Npm
+
+Make sure node 20 is installed, you may use to easly install multiple versions of node [nvm](https://github.com/nvm-sh/nvm),
+use apt or [download node directly](https://nodejs.org/en/download).
+
+## Install Rust
+
+Follow the sets to [install Rust using rustup](https://www.rust-lang.org/tools/install).
```

Comment:
Again: link to which version we use.

Reply:
@horenso Unfortunately, this does not always work. Sometimes a new version emits a warning, old ones didn't etc. Actually, the last upgrade to the latest stable rust version involved quite some effort.

As long as someone takes care that earlier and also the last version are always working, it would be okay. But the much easier variant is to simply support exactly one version and everyone is using this.